README.md

```diff
@@ -4,6 +4,116 @@ CoreTools Healthcheck: [![CoreTools healthcheck](https://azure-functions.visuals
 
 # Dockerhub
 
+## V4 Images
+
+#### Dotnet
+
+`mcr.microsoft.com/azure-functions/dotnet`
+
+Linux amd64 Tags
+
+| Tags                                       | Dockerfile                                                              | OS Version |
+|--------------------------------------------|-------------------------------------------------------------------------|------------|
+| `4`                                      | [Dockerfile](host/4/bullseye/amd64/dotnet/dotnet-inproc/dotnet.Dockerfile)            | Debian 10  |
```

Comment:
@balag0  / @kshyju  - please review - specifically tags, versions and OS version. Thanks!